src/sql/workbench/contrib/query/browser/keyboardQueryActions.ts

```diff
@@ -206,6 +206,29 @@ export class CopyQueryWithResultsKeyboardAction extends Action {
 	}
 }
 
+export class EstimatedExecutionPlanKeyboardAction extends Action {
+	public static ID = 'estimatedExecutionPlanKeyboardAction';
+	public static LABEL = nls.localize('estimatedExecutionPlanKeyboardAction', 'Display Estimated Execution Plan');
```

Comment:
double quote the second parameter.

Reply:
FYI - vs code removed the check for this a while back so we can probably stop asking people to do it one way or another. I'm honestly not even sure what exactly was the reasoning for that in the first place - there isn't any difference between ' and " in JS for strings.